server/models/Types.ts

```diff
@@ -0,0 +1,18 @@
+import { ObjectID } from "mongodb";
```

Comment:
Hey man, for my commit I put the `types.ts` file in `utils/types.ts`. not sure what the pattern is supposed to be, but we should figure it out so we can have a clean merge :]

Reply:
We had the types in `utils/types.ts` last semester and that's where I'd prefer them to be for this semester as well. I think you might have gotten them confused with the Mongoose types, which I can clarify our standup. Essentially, you need to move this file to `utils/types.ts` and Mongoose types will go here since they're used separately.